gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -793,6 +910,73 @@ public static LicenseOption fields(LicenseField... fields) {
   }
 
   /**
+   * Class for specifying operation get options.
+   */
+  class OperationOption extends Option {
+
+    private static final long serialVersionUID = -4572636917684779912L;
+
+    private OperationOption(ComputeRpc.Option option, Object value) {
+      super(option, value);
+    }
+
+    /**
+     * Returns an option to specify the operation's fields to be returned by the RPC call. If this
+     * option is not provided all operation's fields are returned. {@code OperationOption.fields}
+     * can be used to specify only the fields of interest. {@link Operation#operationId()} is
+     * always returned, even if not specified.
+     */
+    public static OperationOption fields(OperationField... fields) {
+      return new OperationOption(ComputeRpc.Option.FIELDS, OperationField.selector(fields));
+    }
+  }
+
+  /**
+   * Class for specifying operation list options.
+   */
+  class OperationListOption extends Option {
+
+    private static final long serialVersionUID = -1509532420587265823L;
+
+    private OperationListOption(ComputeRpc.Option option, Object value) {
+      super(option, value);
+    }
+
+    /**
+     * Returns an option to specify a filter to the operations being listed.
+     */
+    public static OperationListOption filter(OperationFilter filter) {
+      return new OperationListOption(ComputeRpc.Option.FILTER, filter.toPb());
+    }
+
+    /**
+     * Returns an option to specify the maximum number of operations to be returned.
```

Comment:
Should we add "per page" at the end of this sentence?  Also, because the [docs](https://cloud.google.com/compute/docs/reference/latest/globalOperations/list#maxResults) suggest that it's max results per page, should we change the name of this option to "pageSize"?